components/generic/SmartLink.vue

```diff
@@ -56,14 +54,10 @@
         return this.destination;
       },
       isExternalLink() {
-        const destDomain = parseDomain(this.destination);
-        if (destDomain === null) {
-          return false;
-        }
-        const dest = [destDomain.domain, destDomain.tld].join('.');
-        const currentDomain = this.$store.state.request.domain;
-        const current = currentDomain;
-        return dest !== current;
+        const path = this.destination;
+        if (typeof path !== 'string' || path.startsWith('/')) return false;
+        const internalDomain = process.env.INTERNAL_LINK_DOMAIN;
```

Comment:
The env var needs to be documented in /.env.example

Reply:
Let's make `internalDomain` a data property, defaulting to the env var. It will then only need to be set once, and this computed function can then more easily be unit tested.